src/expressions/minExpressionChecker.js

```diff
@@ -1,24 +1,25 @@
 import { BaseSubExpressionChecker } from './baseSubExpressionChecker'
+import { BigUtil } from '../utils/big_util'
 
 export class MinExpressionChecker extends BaseSubExpressionChecker {
   async checkSubExpression (fact, ssid, context) {
-    let hasUndefined = false
     let minResult
     for (const op of fact.operands) {
       const newContext = this._getContextExplainer().extendContextWithExplanation(context)
       const operandResult = await this._getExpressionChecker().checkExpression(op, ssid, newContext)
       this.logger.debug('OperandResult in MIN', operandResult, 'for operand', op)
-      if (typeof minResult === 'undefined' || operandResult < minResult) {
+      if (!BigUtil.isNumeric(operandResult)) {
+        minResult = false
+        break
+      } else if (operandResult === undefined) {
+        minResult = undefined
```

Comment:
Does this work? Doesn't passing undefined into the isNumeric function result in false? Maybe add a test that checks this? We should be able by either doing an integration test checking for potential acts with a MIN expression or directly unit-testing it.